app.js

```diff
@@ -19,14 +21,9 @@ app.on('ready', () => {
     window.setMenu(null);
     window.loadURL('file://' + templates + '/window-main.html');
     window.webContents.on('did-finish-load', function() {
-        const selectedFolders = dialog.showOpenDialog(window, {
-            title: 'Music folder',
-            defaultPath: app.getPath('music'),
-            properties: ['openDirectory']
-        });
-
-        if(selectedFolders !== undefined)
-            folders.add.apply(folders, selectedFolders);
+        global.selectedFolders = config.selectFolders(window, true);
```

Comment:
Again, stay away from global variables. Why is this stored in a global variable if it's used only in this scope?